packages/suite/src/views/wallet/tokens/common/TokensTable/TokensTable.tsx

```diff
@@ -43,26 +27,25 @@ const ChevronIcon = styled(Icon)<{ $isActive: boolean }>`
     transform: ${({ $isActive }) => ($isActive ? 'rotate(0)' : 'rotate(-90deg)')};
 `;
 
-const Header = styled.header<{ $isActive: boolean }>`
+const Header = styled.header<{ $elevation: Elevation; $isActive: boolean }>`
     display: flex;
-    padding: ${spacingsPx.xs};
-    margin: 0 ${spacingsPx.lg};
+    margin: 0 ${spacingsPx.xl};
     cursor: pointer;
     align-items: center;
     ${typography.label}
     color: ${({ theme }) => theme.textSubdued};
-    ${({ $isActive }) =>
+    ${({ $isActive, $elevation, theme }) =>
         $isActive &&
         css`
-            border-bottom: 1px solid ${({ theme }) => theme.borderElevation2};
+            border-bottom: 1px solid ${mapElevationToBorder({ $elevation, theme })};
```

Comment:
you can use just 

```tsx
border-bottom: 1px solid ${mapElevationToBorder};

Reply:
<img width="1024" alt="Screenshot 2024-09-03 at 9 55 13 AM" src="https://github.com/user-attachments/assets/ec0d88c5-d803-4497-b225-70d17dc1971d">

I had it like this before but TS was complaining.